unblob/handlers/__init__.py

```diff
@@ -1,5 +1,17 @@
 from ..models import DirectoryHandlers, Handlers
-from .archive import ar, arc, arj, cab, cpio, dmg, rar, sevenzip, stuffit, tar, zip
+from .archive import (
+    ar,
+    arc,
+    arj,
+    cab,
+    cpio,
+    dmg,
+    rar,
+    sevenzip,
+    stuffit,
+    tar,
+    zip,  # noqa: A004
```

Comment:
I am hesitant about this, maybe rename it to `zip_` or some other safe name?

Reply:
hesitant as well here, so I think you just pushed me enough to rename it

Reply:
a `from ... import zip as ziphandler` would be enough imo

Reply:
oh no